.travis.yml

```diff
@@ -4,15 +4,8 @@ dist: trusty
 language: generic
 
 addons:
-  apt:
-    packages:
-    # GHC
-    - libgmp-dev
-    # Selenium
-    - xvfb
-    # Testcafe
-    - fluxbox
```

Comment:
Don't we need these packages anymore?

Reply:
We do, but now we install them only when we need them:

![image](https://user-images.githubusercontent.com/1523306/58045906-25949d80-7b44-11e9-9755-440dedc036f5.png)